client/asset/btc/electrum_client.go

```diff
@@ -60,6 +60,7 @@ type electrumWalletClient interface {
 	GetRawTransaction(ctx context.Context, txid string) ([]byte, error) // wallet method
 	GetAddressHistory(ctx context.Context, addr string) ([]*electrum.GetAddressHistoryResult, error)
 	GetAddressUnspent(ctx context.Context, addr string) ([]*electrum.GetAddressUnspentResult, error)
+	EstimateSendTxFee(ctx context.Context, txHex string, options interface{}, segwit bool) (fee float64, err error)
```

Comment:
This interface is unrelated to the `btc.Wallet` or `asset.Wallet` interfaces.  This can be any function signature.